scss/custom/_breadcrumb.scss

```diff
@@ -0,0 +1,14 @@
+.breadcrumb {
+  .breadcrumb-item {
+    color: $breadcrumb-item-color;
+    text-decoration: underline;
+
+    &.active {
+      text-decoration: none;
+    }
+  }
+
+  li.breadcrumb-item a {
```

Comment:
In general: if you can style a thing only by using a class, try to do it. So you can overwrite it easier when you need to do. Less specific is always best.

So in this case try `.breadcrumb-item { ...` first and if it is enough, be happy :)